tutorials/FSS.ipynb

```diff
@@ -0,0 +1,2329 @@
+    "        scenario=range(len(input_scenarios))\n",
+    "    ),\n",
+    "    attrs=dict(\n",
+    "        description=\"Fractions skill score\",\n",
+    "    ),\n",
+    ")\n",
+    "da"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "04c11724-e10e-49e3-af69-3ef6f8b96ffd",
+   "metadata": {},
+   "source": [
+    "## Aggregated FSS\n",
+    "\n",
+    "The aggregated FSS groups a list of 2-D spatial fields from the input data array and applies the single 2D FSS in the previous section over each field. It then accumulates the decomposed scores into the final score. A description of the algorithm is provided at the bottom of this tutorial.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1f8aca2d-b043-4551-92e8-1829307d11ab",
+   "metadata": {},
+   "source": [
+    "**1. Setup** \n",
+    "\n",
+    "Similar to the previous section lets start by setting up our inputs. This time we will include the scenarios within the input source itself. Let's define our inputs:\n",
+    "\n",
+    "```\n",
+    "Fcst\n",
+    "----\n",
+    "Dims: T x L x N x M\n",
+    "\n",
+    "Obs\n",
+    "---\n",
+    "Dims: T x N x M\n",
+    "\n",
+    "where,\n",
+    "  T = time_idx [0..24]\n",
+    "  L = lead_time_idx [0..6]\n",
+    "  N = lat_idx [0..400]\n",
+    "  M = lon_idx [0..600]\n",
+    "```\n",
+    "\n",
+    "We will also assume that the variance and bias increase as the lead time increases\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "e35534a7-d8b2-418d-9319-96a160ae3f7c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import xarray as xr\n",
+    "\n",
+    "# lets start by defining a function to generate a T x N x M array since this will be commonly used\n",
+    "T = 24\n",
+    "L = 6\n",
+    "N = 400\n",
+    "M = 600\n",
+    "# set seed for reproducibility\n",
+    "np.random.seed(42)\n",
+    "def _generate_input(dims, mu, sigma):\n",
+    "    return np.random.normal(loc=mu, scale=sigma, size = dims)\n",
+    "\n",
+    "# let's generate obs using the standard normal distribution N(0,1) for a T x N x M field\n",
+    "obs = _generate_input((T, N, M), 0.0, 1.0)\n",
+    "\n",
+    "# now for the fcst we'll need to generate and stack the leadtime arrays\n",
+    "# since we only have 6 lead times we can be a bit lazy and use a for loop\n",
+    "# we will also increase the bias and variance as we increase the leadtime\n",
+    "var_inc = 0.5\n",
+    "bias_inc = 0.25\n",
+    "var, bias = 1.0, 0.0\n",
+    "fcst = []\n",
+    "for i in range(L):\n",
+    "    var += var_inc\n",
+    "    bias += bias_inc\n",
+    "    fcst.append(\n",
+    "        _generate_input((T, N, M), bias, var)\n",
+    "    )\n",
+    "fcst = np.stack(fcst)    \n",
+    "\n",
+    "# we now have the arrays in raw numpy format, we'll need some annotations\n",
+    "# lets convert it to xarray and check that our final dimensions match\n",
```

Comment:
The functions that take a `xarray.DataArray` input should be `xarray`, the single field option doesn't take an `xarray` input because it doesn't (shouldn't) need to have that dependency.